Remotion/Web/Development.WebTesting/ScreenshotCreation/BrowserContentLocators/EdgeBrowserContentLocator.cs

```diff
@@ -0,0 +1,131 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Threading;
+using System.Windows;
+using System.Windows.Automation;
+using JetBrains.Annotations;
+using OpenQA.Selenium;
+using Remotion.Web.Development.WebTesting.Utilities;
+
+namespace Remotion.Web.Development.WebTesting.ScreenshotCreation.BrowserContentLocators
+{
+  /// <summary>
+  /// Locates the browser content area for Edge.
+  /// </summary>
+  public class EdgeBrowserContentLocator : IBrowserContentLocator

```

Comment:
How much of this logic is identical to the ChromeBrowserContentLocator.

Disclaimer: did not check if the window finding a) works and b) could be simplified. Am assuming this was already reviewed / forked from exisitng content locators :)

Reply:
Not a significant amount. I intentionally dropped the browser window rating, as I could not see benefits. The automation tree structure also looks different now, therefore the logic had to be adapted.